src/Components/PaperEditor/TranscriptsContainer/Paragraphs/AnnotationOverlayTrigger.js

```diff
@@ -9,70 +9,88 @@ import {
   faPen,
   faStickyNote,
   faTrashAlt,
-  faTag
+  faTag,
 } from '@fortawesome/free-solid-svg-icons';
 
 const AnnotationOverlayTrigger = (props) => {
+  const {
+    handleEditAnnotation,
+    handleDeleteAnnotation,
+    annotation,
+    labels,
+    words,
+  } = props;
+  const { labelId, id, note } = annotation;
+  const [ label, setLabel ] = useState('');
+  const [ color, setColor ] = useState('');
```

Comment:
`color` and `label` became states, because that is used to determine render. it was also changing after retrieving `labels` in useEffect.